gazelle/internal/swift/rules_common.go

```diff
@@ -14,16 +14,17 @@ func rulesForLibraryModule(
 	srcs []string,
 	swiftImports []string,
 	shouldSetVis bool,
+	swiftLibraryTags []string,
 	buildFile *rule.File,
 ) []*rule.Rule {
 	name, moduleName := ruleNameAndModuleName(buildFile, LibraryRuleKind, defaultName, defaultModuleName)
 	r := rule.NewRule(LibraryRuleKind, name)
 	setCommonSwiftAttrs(r, moduleName, srcs, swiftImports)
 	setVisibilityAttr(r, shouldSetVis, []string{"//visibility:public"})
-	// Mark swift_library targets as manual. We do this so that they are always
-	// built from a leaf node which can provide critical configuration
-	// information.
```

Comment:
Can you put this comment by the `swiftLibraryTags` code in `config.go`? I want to ensure that anyone looking at the code understands why `manual` tags are recommended.